src/reaction_network.jl

```diff
@@ -479,7 +483,7 @@ function massaction_ratelaw_deriv(spec::Symbol, scoef::Int, rate::ExprValues, su
     denom = one(scoef)
     for sub in subs
         name = sub.reactant
-        stoich = sub.stoichiometry        
```

Comment:
Please do "clean up" PRs separate from big changes...

Reply:
You are ofcourse right. Will avoid this in the future.